### PR TITLE
Adding document about Sonarqube

### PR DESCRIPTION
**Pull request by hlong24082k**
I contribute document and usage about SonarQube, which provides guildes for all deverloper to uderstand and handle issue, bug, warning and so on.

### CODE REVIEW

**Comment by ThanhLV97** [comment]
Please give more information about this pull request. @hlong24082k

File: README.md [hunk 1]
```diff
@@ -1,2 +1,77 @@
...
+    docker run -d --name sonarqube -e SONAR_ES_BOOTSTRAP_CHECKS_DISABLE=true -p 9000:9000 sonarqube:latest
+    ```
+
+    >Please waiting a few minutes, Log in to http://localhost:9000 using System Administrator credentials:
+    >- login: admin
+    >- password: admin
+
+    </br>
+
+    ![Login-page](assets/login-page.jpg)
```
Comment (ThanhLV97) [inline README.md]: Naming convetion for this file is not suitable in linix system, Please using underscore

File: README.md [hunk 2]
```diff
@@ -1,2 +1,77 @@
...
+1. Installing SonarQube locally on Linux by entering the following commands in to terminals:
+
+    1. Pull Sonarqube image 
+    ```
+    docker pull sonarqube
+    ```
+
+    2. Start the server by running:
+    ```
+    docker run -d --name sonarqube -e SONAR_ES_BOOTSTRAP_CHECKS_DISABLE=true -p 9000:9000 sonarqube:latest
```
Comment (ThanhLV97) [inline README.md:25]: How about this SONAR_ES_BOOTSTRAP_CHECKS_DISABLE working, I required giving more other flag

**Comment by ThanhLV97** [comment]
Please take a double checking process in the linux server before pushing pull request

File: README.md (hunk 2 above)
Comment (hlong24082k) [reply README.md:25]: I will check information about it.

**Comment by ThanhLV97** [comment]
Tks for your contribution @hlong24082k

**Pull request merged**